bin/slim

```diff
@@ -17,5 +17,11 @@ if (file_exists(__DIR__ . '/../../../autoload.php')) {
     require __DIR__ . '/../vendor/autoload.php';
 }
 
-$app = new Application();
+//Config SHOULD be placed by init command in the project's root directory
+$configDir = dirname(dirname(__DIR__));
```

Comment:
In the case where we use Slim Console as a dependency in project, `dirname(dirname(__DIR__))` will return the correct project root, but what if we use Console as a global dependency? It will return `~/.composer`. Can we make this part dynamic?